google/resource_compute_instance_template.go

```diff
@@ -131,14 +133,6 @@ func resourceComputeInstanceTemplate() *schema.Resource {
 				ForceNew: true,
 			},
 
-			"automatic_restart": &schema.Schema{
```

Comment:
Removals are done by using the `Removed` property and not removing the field entirely, so that any users that are left see a descriptive error and not one that looks like a syntax error.